modules/cli/src/main/java/org/apache/ignite/cli/builtins/node/NodeManager.java

```diff
@@ -111,8 +111,7 @@ public RunningNode start(String nodeName, Path logDir, Path pidsDir, Path srvCfg
             Process p = pb.start();
 
             try (var spinner = new Spinner(out, "Starting a new Ignite node")) {
-
-                if (!waitForStart("Apache Ignite started successfully!", logFile, NODE_START_TIMEOUT, spinner)) {
+                if (!waitForStart("Apache Ignite started successfully!", logFile, p, NODE_START_TIMEOUT, spinner)) {
                     p.destroyForcibly();
 
                     throw new IgniteCLIException("Node wasn't started during timeout period "
```

Comment:
I think it would be nice to add an additional message here about checking log files, what do you think?

Reply:
yep, make sense to me.